### PR TITLE
Fixed koala-migrate script

### PR DESCRIPTION
**Pull request by TobiasDeBruijn**
See title

### CODE REVIEW

File: koala-migrate.py [hunk 1]
```diff
@@ -8,10 +8,19 @@ def get_koala_connection():
     db_password = os.getenv("KOALA_DB_PASSWORD")
     db_host = os.getenv("KOALA_DB_HOST")
     db_port = os.getenv("KOALA_DB_PORT")
-    return psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+
+    conn =  psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+    return conn
```
Comment (Phavion) [inline koala-migrate.py:13]: Ik zie de meerwaarde hier niet van.

File: koala-migrate.py [hunk 2]
```diff
@@ -8,10 +8,19 @@ def get_koala_connection():
...
 def get_mongoose_connection():
-    return psycopg2.connect(f"postgresql:///undead_mongoose")
+    db_name = os.getenv("DB_NAME")
+    db_user = os.getenv("DB_USER")
+    db_password = os.getenv("DB_PASSWORD")
+    db_host = os.getenv("DB_HOST")
+    db_port = os.getenv("DB_PORT")
+
+    conn = psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+    return conn
```
Comment (Phavion) [inline koala-migrate.py:23]: The database name was changed.
There is a good chance that `return psycopg2.connect(f"postgresql:///mongoose")` may fix the problem.

File: koala-migrate.py (hunk 1 above)
Comment (Mstiekema) [reply koala-migrate.py:13]: Mee eens

**Comment by Mstiekema** [comment]
Migrate has been updated in the master branch. Has also been executed on the production server.

**Pull request closed without merging**